CMakeSettings.json

```
@@ -0,0 +1,26 @@
+﻿{
+  "configurations": [
+    {
+      "name": "x64-Debug",
+      "generator": "Ninja",
+      "configurationType": "Debug",
+      "inheritEnvironments": [ "msvc_x64_x64" ],
+      "buildRoot": "${projectDir}\\build\\${name}",
+      "installRoot": "${projectDir}\\out\\install\\${name}",
```

Does this refer to project binaries? I'd rather keep them away from the source.
Let's try
```suggestion
      "buildRoot": "${projectDir}\\build\\debug\\${name}",
      "installRoot": "${projectDir}\\build\\out\\install\\${name}",
```